IHP/WebSocket.hs

```diff
@@ -42,6 +42,9 @@ class WSApp state where
     onClose :: (?state :: IORef state, ?context :: ControllerContext, ?applicationContext :: ApplicationContext, ?modelContext :: ModelContext, ?connection :: Websocket.Connection) => IO ()
     onClose = pure ()
 
+    connectionOptions :: WebSocket.ConnectionOptions
```

Comment:
Can you add the example you added to the PR description here as a haddock comment? So that it will show up in the API docs then